lib/src/Uncertainty/Algorithm/Simulation/openturns/NAISResult.hxx

```diff
@@ -43,32 +43,71 @@ public:
   /** Virtual constructor */
   NAISResult * clone() const override;
 
-  /** Get Samples */
-  Sample getAuxiliarySample() const;
-
-  /** Set NAISResult Samples */
-  void setAuxiliarySample(const Sample & sample);
+  /** Get Input Samples */
+  Sample getAuxiliaryInputSample() const;
+  
+  /** Set Auxiliary input sample */
+  void setAuxiliaryInputSample(const Sample & auxiliaryInputSample);
+  
+  /** Get Output Samples */
+  Sample getAuxiliaryOutputSample() const;
+  
+  /** Set Auxiliary output sample */
+  void setAuxiliaryOutputSample(const Sample & auxiliaryInputSample);
 
   /** Get Auxiliary Density */
   Distribution getAuxiliaryDistribution() const;
-
+  
   /** Set Auxiliary Density */
   void setAuxiliaryDistribution(const Distribution & auxiliaryDistribution);
   
+  /** Set Maximum Outer Sampling */
+  void setMaximumOuterSampling(const UnsignedInteger & maximumOuterSampling);
+  
+  /** Get Maximum Outer Sampling */
+  UnsignedInteger getMaximumOuterSampling() const;
+  
+  /** Set Block Size */
+  void setBlockSize(const UnsignedInteger & blocSize);
+  
+  /** Get Block Size */
+  UnsignedInteger getBlockSize() const;
+  
+  /** Set Variance Estimate */
+  void setVarianceEstimate(const Scalar & varianceEstimate);
+  
   /** Method save() stores the object through the StorageManager */
   void save(Advocate & adv) const override;
 
   /** Method load() reloads the object from the StorageManager */
   void load(Advocate & adv) override;
+  
+  // Not implemented
+  void drawImportanceFactors() const ;
+  void getConfidenceLength() const ;
```

Comment:
overloaded methods should have the same signature, and use the "override" keyword